docs/articles/documentation/test-api/actions/select-text.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: docs
+title: Select Text
+permalink: /documentation/test-api/actions/select-text.html
+---
+# Select Text
+
+There are different kinds of the Select Text action for selecting regular text, `<textarea>` and `contentEditable` content.
+
+* [Select Regular Text](#select-regular-text)
+* [Select \<textarea\> Content](#select-textarea-content)
+* [Perform Selection within Editable Content](#perform-selection-within-editable-content)
+
+## Select Regular Text
+
+```text
+t.selectText( selector, startPos, endPos )
+```
+
+Parameter  | Type                                              | Description
+---------- | ------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------
+`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the webpage element whose text will be selected. See [Selecting Target Elements](index.md#selecting-target-elements).
+`startPos` | Number                                            | The start position of the selection.
+`endPos`   | Number                                            | The end position of the selection.
```

Comment:
I feel like it's good to put in options things that enable a specific mode or provide additional information. Or whose default value is used in most cases.
Things like selection position are not additional, they are essential. They just have their default values which are not expected to be used more often than specific values, I guess.